hw/ip/rv_dm/data/rv_dm.hjson

```diff
@@ -103,15 +103,146 @@
+          desc: "Write to this address to acknowledge that the core is resuming non-debug operation.",
+          count: "NrHarts"
+          swaccess: "wo",
+          hwaccess: "hro",
+          fields: [
+            { bits: "0",
+              resval: "0"
+            },
+          ]
+        }
+      },
+      { name: "EXCEPTION",
+        desc: "An exception was triggered while the core was in debug mode.",
+        swaccess: "wo",
+        hwaccess: "hro",
+        fields: [
+          { bits: "0",
+            resval: "0"
+          },
+        ]
+      },
+      { skipto: "0x300" }
+      { name: "WHERETO",
+        desc: "TODO: No description provided in the spec.",
+        swaccess: "ro",
+        hwaccess: "hrw",
+        fields: [
+          { bits: "31:0",
+            resval: "0"
+          },
+        ]
+      },
+      { skipto: "0x338" }
+      { multireg: {
+          cname: "ABSTRACTCMD"
+          name:  "ABSTRACTCMD"
+          desc:  "TODO: No description provided in the spec."
+          count: "10"
+          swaccess: "ro"
+          hwaccess: "hro"
+          fields: [
+            { bits: "31:0"
+              resval: "0"
+            }
+          ]
+          tags: [// TODO: It is unclear how to predict these values.
+                 "excl:CsrAllTests:CsrExclCheck"]
+        }
+      },
+      { multireg: {
+          cname: "PROGRAM_BUFFER"
+          name:  "PROGRAM_BUFFER"
+          desc:  "TODO: No description provided in the spec."
+          count: "8"
+          swaccess: "ro"
+          hwaccess: "hro"
+          fields: [
+            { bits: "31:0"
+              resval: "0"
+            }
+          ]
+        }
+      },
+      { multireg: {
+          cname: "DATAADDR"
+          name:  "DATAADDR"
+          desc:  "TODO: No description provided in the spec."
+          count: "2"
+          swaccess: "rw"
+          hwaccess: "hro"
+          fields: [
+            { bits: "31:0"
+              resval: "0"
+            }
+          ]
+          tags: [// TODO: Write-read-check will work after "activating" the debug module via JTAG.
+                 "excl:CsrNonInitTests:CsrExclWriteCheck"]
+        }
+      },
+      { skipto: "0x400" }
+      { multireg: {
+          cname: "FLAGS"
+          name:  "FLAGS"
+          desc:  "TODO: No description provided in the spec."
+          count: "256"
+          swaccess: "ro"
+          hwaccess: "hro"
+          fields: [
+            { bits: "31:0"
+              resval: "0"
+            }
+          ]
+        }
+      },
+      // ROM size (given as `items` below) must be a power of two.
       { window: {
-          name: "MEM"
-          items: "1024" // 4 KiB
-          swaccess: "rw",
-          byte-write: "true",
-          desc: '''Access window into the debug memory.'''
+          name: "ROM"
+          items: "512" // 2 KiB
+          swaccess: "ro",
+          desc: '''Access window into the debug ROM.'''
         }
-      }
+      },
```

Comment:
What is currently missing from the debug memory map is the `HaltAddress`, `ResumeAddress` and `ExceptionAddress` following at 0x800 - 0x808. But I guess you're aware of that. Would it make sense to mention that in the hjson somewhere?

Reply:
Good catch @vogelpi - let's add these in a separate PR so that we can make progress with tests that depend on this change.

Reply:
Actually, to clarify: these addresses are the first words inside the debug ROM, an they are thus already exposed via the window here. The debug ROM window starts at byte offset `0x400 + 256 x 4 = 0x800` and exposes a region of another `0x800` bytes which is more than enough to cover the `19 x 8 = 0x98` bytes currently allocated inside the debug ROM:
https://github.com/lowRISC/opentitan/blob/8b1c3bd1c8bf2d8194b4e966fc7de77ad55ee5fe/hw/vendor/pulp_riscv_dbg/debug_rom/debug_rom.sv#L24-L47

But I can add a comment mentioning this in a follow up (don't want to re-trigger CI here ;)).